GameFramework/RotationValue/RotationValue.h

```diff
@@ -0,0 +1,62 @@
+﻿//----------------------------------------------------------
+// <filename>RotationValue.h</filename>
+//----------------------------------------------------------
+
+#ifndef ROTATION_VALUE_H
+#define ROTATION_VALUE_H
+
+#include <d3dx9.h>
+
+/// <summary>
+/// 基礎構築をに関するものをまとめた名前空間
+/// </summary>
+namespace gameframework
+{	
+	/// <summary>
+	/// 角度の基底クラス
+	/// </summary>
+	class RotationValue
+	{
+	public:
+		/// <remarks>
+		/// 範囲は円を一周以上していてはいけない 例:-370度, -2.5 * pi等</br>180度を設定すると最大値が一周するのでこれも不可
+		/// </remarks>
+		RotationValue(float amount, float min, float oneCycleAmount) 
+			:m_oneCycleAmount(oneCycleAmount), m_min(min), m_max(min + oneCycleAmount)
+		{
+			m_rawAmount = amount;
+
+			Normalize(amount);
+		}
+
+		virtual ~RotationValue() =0;
```

Comment:
```suggestion
		virtual ~RotationValue() = 0;
```
どこにスペース入れるかは統一しておきましょう。

Reply:
了解です
修正しました

Reply:
OKです。デストラクタを純粋仮想にする意味がよくわかりませんが…
(最近、僕の知らないC++の機能が多過ぎて困っています。何か特別な意味があるならまた教えて下さい…)